docs/photoniq/event-delivery/event-delivery-filters.md

```diff
@@ -0,0 +1,165 @@
+- `queries`: An array of SELECT statements defining the specific events to filter.
+
+The `SELECT` syntax follows standard SQL conventions, allowing for conditions (`WHERE`), logical operators (`AND`, `OR`), and specifying particular fields or using `*` for all fields within a collection or stream.
+
+**Example JSON filter structure**
+
+```json
+{
+  "action": "add",
+  "once": "FALSE",
+  "initialData":"FALSE",
+  "queries": [
+    "select * from CollectionName where condition1",
+    "select fieldName from CollectionName where condition2 OR condition3"
+  ]
+}
+```
+
+
+## Filter use cases and examples
+
+Using event delivery queries allows for highly customizable data streams tailored to various application needs. Whether monitoring real-time updates from a specific dataset or filtering changes based on specific criteria, these queries ensure your application processes only the most relevant information.
+
+This section demonstrates how to subscribe to and filter events using practical examples. You can interact with the PhotonIQ Event Delivery Service using `wscat` for WebSockets or `curl` for SSE. These examples highlight the flexibility of event delivery queries in EDS, allowing precise control over the data stream with SQL-like syntax and dynamic filter management to meet your specific requirements.
+
+
+### Queries
+
+To subscribe to events using EDS, send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). It requires the following fields:
+
+- **EDS host**: The host where the EDS service is running.
+- **x-customer-id**: The is used to authenticate the request.
+- **type**: Type of subscription (e.g., `collection`).
+- **filters**:Fields to specify the desired event criteria.
+
+:::important
+Contact your Macrometa patners for your EDS host and `x-customer-id`
+
+:::
+
+<Tabs groupId="operating-systems">
+  <TabItem value="ws-syntax" label="WebSockets">
+
+Curl currently has no support for WebSockets, you can use [wscat](https://github.com/WebSockets/wscat) to send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get) via WebSockets as shown below:
+
+```bash
+  wscat -c 'wss://<eds-host>/api/es/v1/subscribe?type=collection&x-customer-id=<x-customer-id>&filters={"action": "add", "once": "FALSE", "queries": ["select fieldName from CollectionName"]}'
+```
+
+  </TabItem>
+  <TabItem value="sse-syntax" label="Server-Sent Events">
+
+Here is the syntax to send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get) via SSE:
+
+```bash
+  curl -X POST -H "Content-Type: application/json" -H "x-customer-id: <x-customer-id>" -d '{"type": "collection", "filters": {"once": "TRUE", "compress": "FALSE", "initialData":"TRUE", "queries": ["select fieldName from CollectionName"]}}' https://<eds-host>/api/es/sse/v1/subscribe
```

Comment:
In SSE, the auth key is provided in the header like customer id. E.g  -H "Authorization: <apiKey>"